andela/hc-beef-sauce
# #158532832 Nag User when Check is Down

#### What does this PR do?
If a user has the nag feature turned on, they will periodically receive an email when their check is down. The time between nagging emails is the "GRACE period + TIMEOUT period". 

#### Description of Task to be completed?
Have the following tasks working in the system:

- A user can set whether a specific check has nag mode enabled
- If the nag mode is true, the user will get an email periodically until the check is up or they turn off nag mode
- The time between nags is set to Grace period + Timeout Period
- A special icon should display when the nag period has been entered

#### How should this be manually tested?
- Create a check and ping it
- Enable nag mode on the check
- Give it a timeout and grace period of 1 minute each
- If no ping is sent again, the alerts continue to come in intervals of 2 mins

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/158532832

#### Screenshots (if appropriate)
